### PR TITLE
Add ukrainian (uk) translation

### PR DESCRIPTION
**Pull request by baytuch**

### CODE REVIEW

**Comment by appgurueu** [comment]
Not sure, but shouldn't the ISO language code for this be `ua`?

**Comment by baytuch** [comment]
I think, must be UK

**Comment by appgurueu** [comment]
> I think, must be UK

Yeah you're right.

**Comment by baytuch** [comment]
I added a few more changes in translation to improve it.

**Review by sfan5 — APPROVED** [review]
Looks good.

File: mods/bones/locale/bones.uk.tr [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# textdomain: bones
+Bones=Кістки
+@1's old bones=Старі кістки @1
+@1 died at @2.=@1 помер в @2.
+@1 died at @2, and dropped their inventory.=@1 помер в @2 та втратив вміст свого інвентарю.
```
Comment (appgurueu) [inline mods/bones/locale/bones.uk.tr:5]: Google Translate translates this as "... and lost the contents of his inventory ...", which might be misleading (dropping = turned to items, lost = gone forever)?

**Review by appgurueu — APPROVED** [review]
Minor "nuance" question, but seems about right.

File: mods/bones/locale/bones.uk.tr (hunk 1 above)
Comment (baytuch) [reply mods/bones/locale/bones.uk.tr]: I clarified the translation of this proposal not to create inaccuracies. You are right, gamer can interpret this  ambiguously.

**Pull request merged**